Update WND and Kerberos (#1209)
owncloud/docs-server: modules/admin_manual/pages/configuration/server/config_apps_sample_php_parameters.adoc

be for a service account with special privileges, in particular, it must
be able to impersonate the users. It is highly recommended that the password
for this service account doesn't expire, otherwise you will have to replace
the file manually before the expiration. See the Kerberos documentation for details.

- `ocservice` (required): The name of the service of the account. This matches

> the SPN of the Windows / Samba account. It usually is in the form "HTTP/<hostname>",

Author: Sourced from core

"user001@my.dom.com" will be mapped to the same windows user as
"user001@not.mine.eu".

- `EALdapAttr`: Use ownCloud's user extended attributes to map the ownCloud
user to the target LDAP attribute. The mapping has an "attr" parameter in order
to select the LDAP attribute you want to use. Note that it's required that
the user_ldap app exposes the chosen attribute which requires user_ldap 0.19.0+.

> In case ownCloud can't map the user, an error will be thrown.

Reviewer: ```suggestion
If ownCloud can't map the user, an error will be thrown.
```

Author: Same as below,will do in core and transport it back

Reviewer: This is not a really important change, so don't worry too much about having to push another cocre PR.

Author: already in pushing state... 😄

Author: pls approve/merge: https://github.com/owncloud/core/pull/41134
